GUIDELINES.md

```diff
@@ -171,6 +171,18 @@ Example:
 }
 ```
 
+### User Permissions
+
+#### Fetching objects
+
+When fetching objects if the User has permissions to fetch it but he lacks a specific property permission then that property will not be returned in the response.
```

Comment:
Would also not split up the reading and writing since they're both related to restricted properties.

```suggestion
Sometimes users don't have the required permissions to read or write certain data. In our API responses, we should not return the properties the user has no access to.

If that user tries to change a property he is not allowed to change, then we will return a `403` error response with an appropriate explanation.
```